src/types/sigstore/index.ts

```diff
@@ -120,25 +121,32 @@ export function isVerifiableTransparencyLogEntry(
   );
 }
 
-export const bundle = {
-  toDSSEBundle: (
-    envelope: Envelope,
-    signature: SignatureMaterial,
-    rekorEntry: Entry
-  ): Bundle => ({
+export function toDSSEBundle({
+  envelope,
+  signature,
+  tlogEntry,
+}: {
+  envelope: Envelope;
+  signature: SignatureMaterial;
+  tlogEntry?: Entry;
+}): Bundle {
```

Comment:
Switch from ordered arguments an object so we can make the `tlogEntry` optional (and add support for an optional `timestampEntry` in the future)